Observer/ConfigReview.php

```diff
@@ -134,4 +143,17 @@ private function _reviewAddressValidation()
             // @codingStandardsIgnoreEnd
         }
     }
+
+    /**
+     * @return void
+     * @SuppressWarnings(Generic.Files.LineLength.TooLong)
+     */
+    private function _reviewSandboxMode()
+    {
+        if ($this->taxjarConfig->isSandboxEnabled()) {
+            // @codingStandardsIgnoreStart
+            $this->messageManager->addWarningMessage(__('Sandbox mode is enabled - this mode should only be used for testing.  Some features may not work as expected.'));
```

Comment:
Think we should be clearer here around sandbox for calculations and staging environments:

```
Sandbox mode is enabled for sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected.
```

Can we add HTML here to link to our sandbox environment docs?

```html
Sandbox mode is enabled for sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected. <a href="https://developers.taxjar.com/api/reference/#sandbox-environment" target="_blank" rel="noopener">Learn more</a>
```